### PR TITLE
Typo codingrule

### PR DESCRIPTION
**Pull request by JeongUiSeong**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [6815498ccad1fd04da8ca48f7633351be5940778] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6815498ccad1fd04da8ca48f7633351be5940778] - Code Rule Check (C++) OK.

File: apps/examples/hello/hello_main.c [hunk 1]
```diff
@@ -67,6 +67,6 @@ int main(int argc, FAR char *argv[])
 int hello_main(int argc, char *argv[])
 #endif
 {
-	printf("Hello, World!!\n");
+	printf("Hello, Worlfd!!\n");
```
Comment (junmin-kim) [inline apps/examples/hello/hello_main.c]: I think this is typo

**Comment by sunghan-chang** [comment]
@JeongUiSeong Thank you for contribution. But this PR is incorrect. Let's close this now. When you re-edit this, please re-open this.

**Pull request closed without merging**